File: week03/week03-4.py
#week03-4.py
#LeetCode 28. Find the Index of the First Occurrence in a String
#學習計畫第3題 Easy 在一堆草堆裡haystack，找到一支針needle
#在Python只要一行 a.find(b) 在字串 a 裡找到 字串 b
class Solution:
    def strStr(self, a: str, b: str) -> int:
        return a.find(b)

File: week03/week03-4.cpp
//week03-4.cpp 學習計畫第3題 大海撈針 在 haystack 找needle
//LeetCode 28. Find the Index of the First Occurrence in a String
class Solution {
public:
    int strStr(string haystack, string needle) {
        //https://cplusplus.com/reference/string/string/find/
        return haystack.find(needle);
    }
};